calico-cloud/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -27,18 +27,19 @@ Images must be available locally through the container runtime environment where
 
 ## How to
 
-- [Get latest version of Image Assurance](#get-latest-version-of-image-assurance)
-- [Start the scanner](#start-the-scanner)
-- [Scan images](#scan-images)
+- [Get the latest version of Image Assurance](#get-the-latest-version-of-image-assurance)
+- [Download the scanner](#download-the-scanner)
```

Comment:
In section to download scanner, doc says the following. Isn't the command just to verify the correct version, not that scanner works correctly?

Verify that the scanner works correctly by running the version command.
./tigera-scanner version
v1.5.6